lib/testing/src/lib/content-services/rest-apis/upload/upload-api.ts

```diff
@@ -0,0 +1,69 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { RepoApi } from '../repo-api';
+import { UploadApi as AdfUploadApi } from '@alfresco/js-api';
+
+import * as fs from 'fs';
+
+export class UploadApi extends RepoApi {
```

Comment:
we already have UploadActions this seems a duplication

Reply:
upload-api and upload.actions indeed overlap a bit. Talking with Cristina and Roxana, and from what I saw in another PR, it's best we go with these 2 in parallel for now, and create tasks for changing the test files to move from the old upload.actions to the new upload-api
upload.actions also contains methods belonging to nodes-api, those need to be replaced with new calls and then removed.